Extension/src/LanguageServer/Providers/inlayHintProvider.ts

```diff
@@ -204,7 +203,7 @@ export class InlayHintsProvider implements vscode.InlayHintsProvider {
     private resolveTypeHints(settings: CppSettings, hints: CppInlayHint[]): vscode.InlayHint[] {
         const resolvedHints: vscode.InlayHint[] = [];
         for (const hint of hints) {
-            const showOnLeft: boolean = settings.inlayHintsAutoDeclarationTypesShowOnLeft && hint.identifierLength > 0;
+            const showOnLeft: boolean | undefined = settings.inlayHintsAutoDeclarationTypesShowOnLeft && hint.identifierLength > 0;

```

Comment:
@browntarik settings.inlayHintsAutoDeclarationTypesShowOnLeft isn't typed to return `undefined` so it doesn't seem like this `| undefined` should here?